sdk/ai/azure-ai-generative/azure/ai/generative/synthetic/simulator/simulator/_proxy_completion_model.py

```diff
@@ -29,8 +30,8 @@ def __init__(self, name, template_key, template_parameters, *args, **kwargs):
 
         super().__init__(name=name, *args, **kwargs)
 
-    def format_request_data(self, messages: List[dict], **request_params): # type: ignore[override]
-        request_data = {"messages": messages, **self.get_model_params()}
+    def format_request_data(self, prompt: List[dict], **request_params):  # type: ignore[override]
```

Comment:
Why are we changing the name of the argument from messages to prompt?

Reply:
It is because the parameter has a different name than in the implemented interface or in an overridden method. Originally it is called prompt